pyart/correct/bias_and_noise.py

```diff
@@ -144,6 +145,62 @@ def correct_bias(radar, bias=0.0, field_name=None):
     return corr_field
 
 
+def calc_zdr_offset(radar, gatefilter=None, height_range=None, zdr_var=None):
+    """
+    Function for calculating the ZDR bias from a VPT scan.
+
+    Parameters
+    ----------
+    radar : PyART radar object
+        Radar object with radar data
+    gatefilter: PyART GateFilter
+        Gatefilter for filtering out data for calculating ZDR bias
+    height_range: tuple
+        The minimum and maximum heights in meters for the scan.
+    zdr_var: string or None
+        The name of the ZDR variable. Set to None to have PyART try to determine this automatically.
+
+    Returns
+    -------
+    obj : dict
```

Comment:
Can we rename obj to something else? Not really necessary, but more so i think obj as a class or pyart or xarray obj and not dict